src/Microsoft.NetCore.Analyzers/UnitTests/Security/ApprovedCipherModeTests.cs

```diff
@@ -0,0 +1,108 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.Diagnostics;
+using Test.Utilities;
+using Xunit;
+
+namespace Microsoft.NetCore.Analyzers.Security.UnitTests
+{
+    public class ApprovedCipherModeTests : DiagnosticAnalyzerTestBase
```

Comment:
Also need some negative test cases.  Make sure that code you think is safe doesn't generate diagnostics.